<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openDBDialog</class>
 <widget class="QDialog" name="openDBDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayoutMain">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="HostAdresslineEdit">
         <property name="text">
          <string>217.172.21.199</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DataBaseNamelineEdit">
         <property name="text">
          <string>cddb</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UserNamelineEdit">
         <property name="text">
          <string>student</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UserPasswordlineEdit">
         <property name="text">
          <string notr="true">3265</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="OpenDataBasePushButton">
         <property name="text">
          <string>Open Data Base</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ClosePushButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ClosePushButton</sender>
   <signal>clicked()</signal>
   <receiver>openDBDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
